bitcoin/tx.c

```diff
@@ -228,6 +228,12 @@ const u8 *bitcoin_tx_output_get_script(const tal_t *ctx,
 	return res;
 }
 
+void bitcoin_tx_output_get_amount_sat(struct bitcoin_tx *tx, int outnum,
+				      struct amount_sat *amount)
+{
+	amount->satoshis = tx->wtx->outputs[outnum].satoshi;  /* Raw: type conversion */
+}
+
```

Comment:
This should likely use `bitcoin_tx_output_get_amount`, `assert` that we're dealing with the fee-asset, and then return the converted value in `amount_sat` instead. As it is this'd fail attempting to access `.satoshi` on elements since that field is not populated.